src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -48,6 +62,10 @@ public function getUpgradeStrategy(): string
     }
 
     /**
+     * Specification:
```

Comment:
- Defines the default source code provider.
- Available options: GitHub (default) and GitLab.

Reply:
++